### PR TITLE
Swag update

### PR DESCRIPTION
**Pull request by migetman9**
An update for the MAGWest Swag. 

NOT READY TO MERGE

### CODE REVIEW

**Comment by RobRuana** [comment]
Just ping me whenever you need me!

File: magwest/templates/static_views/tiara.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>Tiara</h1>
+
+<p>
+  Take note! These tiara's are pretty, and pretty fragile too. Don't try to bend them!
```
Comment (ShevaDas) [inline magwest/templates/static_views/tiara.html]: *tiaras

File: magwest/templates/static_views/pin.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <title>Pin</title>
+  <style type="text/css">
+    html, html * { font-family: Arial, Helvetica, sans-serif; }
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>Oh my word this is so metal! pin.</h1>
```
Comment (ShevaDas) [inline magwest/templates/static_views/pin.html]: maybe "Oh my word, this is so metal (pin)!"

File: magwest/templates/static_views/pin.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<p>
+  Stick to the man. Stick to your friends. Stick it on your shirt. Its a metal pin with the MAGWest event logo.
```
Comment (ShevaDas) [inline magwest/templates/static_views/pin.html]: *It's

File: magwest/templates/static_views/lanyard.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  <title>LANYard</title>
+  <style type="text/css">
+    html, html * { font-family: Arial, Helvetica, sans-serif; }
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>What do you call three feet of ethernet cable? A LANYard.</h1>
```
Comment (ShevaDas) [inline magwest/templates/static_views/lanyard.html]: *LANyard.

Comment (migetman9) [inline magwest/templates/static_views/lanyard.html]: Disagree

**Comment by RobRuana** [comment]
Good to go?

**Comment by ShevaDas** [comment]
Should be! My PR in the other repo needs to be merged simultaneously, I
think.

**Pull request merged**